lib/graph/helpers/ui5Framework.js

```diff
@@ -369,7 +369,7 @@ export default {
 			cwd: rootProject.getRootPath(),
 			version,
 			providedLibraryMetadata,
-			...options.resolverConfig,
+			cacheMode: options.cacheMode
```

Comment:
_Minor_: Could be destructured in line 291 (like `workspace´).

Reply:
Done